sled-agent/src/instance_manager.rs

```diff
@@ -335,12 +335,12 @@ impl InstanceManager {
 enum InstanceManagerRequest {
     EnsureRegistered {
         propolis_id: PropolisUuid,
-        instance: InstanceEnsureBody,
         // These are boxed because they are, apparently, quite large, and Clippy
         // whinges about the overall size of this variant relative to the
         // others. Since we will generally send `EnsureRegistered` requests much
         // less frequently than most of the others, boxing this seems like a
         // reasonable choice...
+        instance: Box<InstanceEnsureBody>,
```

Comment:
take it or leave it: perhaps, instead of boxing the `InstanceEnsureBody` and the `SledIdentifiers` separately, we should just take the whole variant and stick it inside one big box? Not performance critical, so this doesn't actually matter all that much, but it might feel a little nicer?

Reply:
I think I will leave it, as to me it feels weird to box the entire variant.

Another option could be to do `Box<(InstanceEnsureBody, SledIdentifiers)>`. But I don't think it matters that much either way.